src/components/login/SignIn.tsx

```diff
@@ -7,27 +7,37 @@ import {
   InputGroup,
   InputLeftElement,
   Text,
-} from '@chakra-ui/react';
-import { AxiosError } from 'axios';
-import { useState } from 'react';
-import { AiFillLock, AiOutlineMail } from 'react-icons/ai';
-import { Flex } from '../primitives/Flex';
-import { useAuth } from '../../contexts/auth';
-import { theme } from '../../theme';
-import { ChangeLoginMode } from './ChangeLoginMode';
+  useToast,
+} from "@chakra-ui/react";
+import { AxiosError } from "axios";
+import { useState, useRef } from "react";
+import { AiFillLock, AiOutlineMail } from "react-icons/ai";
+import { Flex } from "../primitives/Flex";
+import { useAuth } from "../../contexts/auth";
+import { theme } from "../../theme";
+import { ChangeLoginMode } from "./ChangeLoginMode";
 
 type SignInProps = { changeMode: () => void };
 export const SignIn = ({ changeMode }: SignInProps) => {
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const [error, setError] = useState('');
+  const [email, setEmail] = useState("");
```

Comment:
this is overriding the default linter of the project, pls adjust your vscode to follow the project's pattern, if you need any help lmk